src/land/UberArcanistSubmitQueueEngine.php

```diff
@@ -190,15 +183,28 @@ protected function pushChangeToSubmitQueue() {
     // Get the latest revision as we could have updated the diff
     // as a result of arc diff
     $revision = $this->getRevision();
-    $statusUrl = $this->submitQueueClient->submitMergeRequest(
-      $remoteUrl,
-      $revision['diffs'][0],
-      $revision['id'],
-      $this->shouldShadow,
-      $this->getTargetOnto());
-    $this->writeInfo(
-      pht('Successfully submitted the request to the Submit Queue.'),
-      pht('Please use "%s" to track your changes', $statusUrl));
+
+    if ($this->getSkipSubmitQueueChecks()) {
+      $statusUrl = $this->submitQueueClient->submitPriorityMergeRequest(
+        $remoteUrl,
+        $revision['diffs'][0],
+        $revision['id'],
+        $this->shouldShadow,
+        $this->getTargetOnto());
+      $this->writeInfo(
+        pht('Successfully submitted the priority merge request to the Submit Queue.'),
+        pht('Please use "%s" to track your changes', $statusUrl));
+    } else {
+      $statusUrl = $this->submitQueueClient->submitMergeRequest(
+        $remoteUrl,
+        $revision['diffs'][0],
+        $revision['id'],
+        $this->shouldShadow,
+        $this->getTargetOnto());
+      $this->writeInfo(
+        pht('Successfully submitted the request to the Submit Queue.'),
+        pht('Please use "%s" to track your changes', $statusUrl));
+    }
```

Comment:
Nit: slightly shorter version
```suggestion
$method = $this->getSkipSubmitQueueChecks()
    ? 'submitPriorityMergeRequest'
    : 'submitMergeRequest';

$statusUrl = $this->submitQueueClient->$method(
    $remoteUrl,
    $revision['diffs'][0],
    $revision['id'],
    $this->shouldShadow,
    $this->getTargetOnto()
);

$message = $this->getSkipSubmitQueueChecks()
    ? pht('Successfully submitted the priority merge request to the Submit Queue.')
    : pht('Successfully submitted the request to the Submit Queue.');

$this->writeInfo($message, pht('Please use "%s" to track your changes', $statusUrl));

```

Reply:
Good call, but I actually ended up removing a few of these args from `submitPriorityMergeRequest`, so can't do that anymore.